packages/rocketchat-livechat/config.js

```diff
@@ -66,6 +66,14 @@ Meteor.startup(function() {
 		i18nLabel: 'Offline_success_message'
 	});
 
+	RocketChat.settings.add('Livechat_conversation_finished_message', '', {
+		type: 'string',
+		group: 'Livechat',
+		public: true,
+		section: 'Conversation Finished',
```

Comment:
I don't think this setting should be on a new Section, it can be confusing (there are other settings that might fit here) and I don't see a reason for a section with only one setting. so I would rather put it bellow `Show agent email`  agent email setting 😉

Reply:
Yes, you're totally right.
I made a mistake, to be honest I'm surprise reviewing this code.
Fixed.